react/components/SelectionListOrderBy.js

```diff
@@ -24,6 +24,7 @@ class SelectionListOrderBy extends Component {
   }
 
   static propTypes = {
+    mobile: PropTypes.bool,
```

Comment:
You could get this value from runtime. This component could also be functional, using the useState hook, like this: `const [showDropdown, setShowDropdown] = useState(false)`